examples/rust/src/bin/btc/main.rs

```diff
@@ -1,23 +1,26 @@
 #![allow(unused_imports)]
+#![allow(dead_code)]
+
 use std::collections::HashMap;
 use std::marker::PhantomData;
 use std::path::{Path, PathBuf};
 use std::thread::JoinHandle;
 use std::{env, thread};
 
 use chrono::{DateTime, Utc};
-use raphtory::core::utils;
+use raphtory::core::utils::hashing;
 use raphtory::core::{Direction, Prop};
 use raphtory::graph_loader::source::csv_loader::CsvLoader;
+use raphtory::prelude::*;
 use regex::Regex;
 use serde::Deserialize;
 use std::fs::File;
 use std::io::{prelude::*, BufReader, LineWriter};
 use std::time::Instant;
 
-use raphtory::db::graph::Graph;
-use raphtory::db::mutation_api::AdditionOps;
-use raphtory::db::view_api::*;
+use raphtory::db::api::mutation::AdditionOps;
+use raphtory::db::api::view::*;
+use raphtory::prelude::*;
```

Comment:
Note that prelude is imported twice! The other two imports here should also already be included in the prelude